addons/account/models/res_config_settings.py

```diff
@@ -32,6 +32,22 @@ class ResConfigSettings(models.TransientModel):
     group_warning_account = fields.Boolean(string="Warnings in Invoices", implied_group='account.group_warning_account')
     group_cash_rounding = fields.Boolean(string="Cash Rounding", implied_group='account.group_cash_rounding')
     group_fiscal_year = fields.Boolean(string='Fiscal Years', implied_group='account.group_fiscal_year')
+    # group_show_price_subtotal and group_show_price_total are opposite,
+    # so we can assume exactly one of them will be set, and not the other.
+    # We need both of them to coexist so we can take advantage of automatic group assignation.
+    group_show_price_subtotal = fields.Boolean(
```

Comment:
These 3 fields are already defined in sales/models/res_config.py.

Here you just overwrite them. 

By the way you can't have 2 times the same field on a view, so I'm not sure that what you did really work.

Reply:
Ooops, forget what I said, I didn't see that you removed the fields in sale.

Furthermore, sale depends on account and not the contrary. 

It would have been easier to review this if this simple change was isolation in a single commit.